oauth2client/contrib/appengine.py

```diff
@@ -477,15 +469,15 @@ def _parse_state_value(state, user):
         user: google.appengine.api.users.User, The current user.
 
     Raises:
-        InvalidXsrfTokenError: if the XSRF token is invalid.
+        ValueError: if the XSRF token is invalid.
```

Comment:
Would it be acceptable to have `_parse_state_value()` return `None` for invalid tokens and a string otherwise?  Similar to `oauth2client.contrib.flask_util._get_flow_for_token()`